src/main/java/pages/BasePage.java

```diff
@@ -0,0 +1,40 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class BasePage {
+    public WebDriver driver;
+    public Actions action;
```

Comment:
the `Webdriver` and the `Actions` are better to have the "protected" type access modifier